### PR TITLE
Ursine description

### PR DESCRIPTION
**Pull request by Khama-Llama**
# Developer changelog:
- Rewrote Ursine description.
- Removed both Primitive Shaman lines.

Addresses #1563.

### CODE REVIEW

File: localisation/wc_religions.csv [hunk 1]
```diff
@@ -134,8 +134,6 @@ elemental_lords_group;Elementalist;Elementalist;Elementalist;;;;;;;;;;;x
...
-primitive_shamanism_DESC;The worshipping of primitive godlike figures, as the bear twins Ursoc and Ursol, is uncommon in Azeroth, however, their followers primarily revere nature and the spirits of the world much like "modern" shamanism. The bear twins are the primary patrons of those who believe in a balance between nature and all living creatures. The furbolg are the most fervent followers of the bear twins in the present day, following in their magical tradition and carving their fortress, Timbermaw Hold, in the form of a gigantic, growling bear head.;;Die Verehrung primitiver, gott�hnlicher Gestalten, wie der B�renzwillinge Ursoc und Ursol, ist in Azeroth nicht allzu verbreitet, ihre Anh�nger verehren jedoch in erster Linie die Natur und die Geister der Welt, �hnlich wie der �moderne� Schamanismus. Die B�renzwillinge sind die prim�ren Schirmherren derer, die an ein Gleichgewicht zwischen der Natur und allen Lebewesen glauben. Die Furbolgs sind die gl�hendsten Anh�nger der B�renzwillinge in der heutigen Zeit. Sie folgen ihrer magischen Tradition und schnitzten ihre Festung, die Holzschlundfeste, in Form eines gigantischen, knurrenden B�renkopfes.;;;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/wc_religions.csv:138]: Wait, why they're removed?

Comment (Khama-Llama) [reply localisation/wc_religions.csv:138]: I went on a limb figuring the variety of elemental/spiritual worship is going to patron gods and the like. As shown with the Quilboars, Kobolds, and Furbolgs. With what I know there isn't anything to place in the description when these are getting their own. Is the intention to use this religion to envelope all those and have it used side by side with them?

Comment (zumbak04) [reply localisation/wc_religions.csv:138]: > I went on a limb figuring the variety of elemental/spiritual worship is going to patron gods and the like. As shown with the Quilboars, Kobolds, and Furbolgs. With what I know there isn't anything to place in the description when these are getting their own. Is the intention to use this religion to envelope all those and have it used side by side with them?

The religion is a placeholder for races that don't have any gods and simply worship primitive spirits or elementals (like gnolls).

Comment (Khama-Llama) [reply localisation/wc_religions.csv]: Modifications made.

**Pull request merged**